assets/app/scripts/services/imagestreams.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("ImageStreamsService", function(){
+    function ImageStreamsService() {}
+
```

Comment:
Nice. Now I would just return a hash rather than `new ImageStreamService()` and having to mess with a prototype:

```javascript
angular.module("openshiftConsole")
  .factory("ImageStreamsService", function(){
    return {
       tagsByName: function(imageStream) { /*  */  }
    }
});
```